.github/actions/build-zeebe/action.yml

```
@@ -44,4 +44,4 @@ runs:
         mvn -B -DskipTests -DskipChecks install ${{ inputs.maven-extra-args }}
         export BUILD_DIR=$(mvn -pl dist/ help:evaluate -Dexpression=project.build.directory -q -DforceStdout)
         export ARTIFACT=$(mvn -pl dist/ help:evaluate -Dexpression=project.build.finalName -q -DforceStdout)
-        echo "::set-output name=result::${BUILD_DIR}/${ARTIFACT}.tar.gz"
+        echo "distball=result::${BUILD_DIR}/${ARTIFACT}.tar.gz" >> $GITHUB_OUTPUT
```

should be like this?
```suggestion
        echo "result=${BUILD_DIR}/${ARTIFACT}.tar.gz" >> $GITHUB_OUTPUT
```

---

renaming it to `distball` would also require adjusting line 26

---

Yep, already made the change :+1: